common/persistence/cassandra/queue_v2_store.go

```diff
@@ -61,7 +60,7 @@ const (
 	TemplateGetQueueQuery            = `SELECT metadata_payload, metadata_encoding, version FROM queues WHERE queue_type = ? AND queue_name = ?`
 	TemplateRangeDeleteMessagesQuery = `DELETE FROM queue_messages WHERE queue_type = ? AND queue_name = ? AND queue_partition = ? AND message_id >= ? AND message_id <= ?`
 	TemplateUpdateQueueMetadataQuery = `UPDATE queues SET metadata_payload = ?, metadata_encoding = ?, version = ? WHERE queue_type = ? AND queue_name = ? IF version = ?`
-
+	templateGetQueueNamesQuery       = `SELECT queue_name FROM queues WHERE queue_type =? ALLOW FILTERING`
```

Comment:
As discussed, let's add a comment about this

Reply:
Comment added.